src/locales/zh-Hant.json

```
@@ -652,11 +652,12 @@
   "nft_sold_out": "售罄",
   "nft_stock_left": "僅餘 {stock} 本",
   "nft_supply_section_title": "供應量",
-  "nft_tipping_button_continue": "繼續",
-  "nft_tipping_button_skip": "不用了",
+  "nft_tipping_button_continue": "額外打賞",
+  "nft_tipping_button_loading": "正在導到付款頁...",
+  "nft_tipping_button_skip": "不額外打賞",
   "nft_tipping_placeholder": "或其他自訂金額",
-  "nft_tipping_title": "想給予額外支持嗎？",
-  "nft_tipping_description": "款項將會分配給創作者",
+  "nft_tipping_title": "想額外支持嗎？",

```

It seems the original version is better..
```suggestion
  "nft_tipping_title": "想給予額外支持嗎？",

```